src/game/utils.js

```diff
@@ -54,10 +54,13 @@ export function shuffleCards(ctx, startingCard) {
 export function setupGame(ctx, setupData) {
   const startSuit = (setupData) ? setupData.startSuit || DEFAULT_START_SUIT : DEFAULT_START_SUIT;
   const gameMode = setupData ? setupData.gameMode || DEFAULT_GAME_MODE : DEFAULT_GAME_MODE;
-  const startingCard = STARTING_CARD_MAP[startSuit];
-  // remove invalid cards
-  if (!isGameModeCornucopia(gameMode)) {
+  let startingCard;
+
+  if (isGameModeCornucopia(gameMode)) {
+    startingCard = startSuit + '2';
```

Comment:
I believe it would be better to have the same logic for EoP and Cornucopia, i.e. there should also be a `STARTING_CARD_MAP` for Cornucopia. And maybe even an (empty) `INVALID_CARDS`. It would be better to have consistent solutions.

Reply:
@matthewejones   While you are at it, please take the opportunity and also have a look at https://github.com/dehydr8/elevation-of-privilege/issues/140